src/service/artifact.ts

```diff
@@ -115,16 +119,34 @@ export async function getArtifacts(
             '%2F'
           )}`,
           vulnerabilities: {
-            count: vulnerabilityCount,
-            severity: severity,
-            critical: critical,
-            high: high,
-            medium: medium,
-            low: low,
-            none: none,
+            count: 0,
+            severity: "none",
+            critical: 0,
+            high: 0,
+            medium: 0,
+            low: 0,
+            none: 0,
           },
           id: projectId + generatedTag + element.push_time,
         }
+
+        const reportMimeType = "application/vnd.security.vulnerability.report; version=1.1"
```

Comment:
It would be better if this string came from config.

Reply:
I did not notice that the value can be `application/vnd.scanner.adapter.vuln.report.harbor+json; version=1.0` or `application/vnd.security.vulnerability.report; version=1.1`, thanks